content/sql/concepts/math-functions/terms/atan/atan.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: ATAN Function
+Description: Trigonometric function returning the arctangent angle in SQL
+Subjects:
+  - SQL
+  - Mathematics
+Tags:
```

Comment:
The tags need to be in single quotes and need to come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).